File: 00-node/01-examples/03-get-and-docker/core/book.js
'use strict'

let book = {
    getAllBooks: () => {
        let books = [
            {title: 'Code Complete 2nd Edition', author: 'Steve McConnell', year: '2004'}, 
            {title: 'Clean Code using Docker', author: 'Robert C. Martin', year: '2008'}, 
            {title: 'The Last Lecture', author: 'Randy Pausch', year: '2008'} 
        ];
        
        return books;
    }
};

module.exports = book;

File: 02-netcore/01-examples/04-get-and-exception-handling/Core/Book.cs
using System.Collections.Generic;
using _04_get_and_exception_handling.Core.Models;

namespace _04_get_and_exception_handling.Core
{
    public class Book
    {
        public IEnumerable<BookInformation> GetAllBooks()
        {
            var books = new List<BookInformation>();
            books.Add(new BookInformation("Code Complete 2nd Edition", "Steve McConnell","2004"));
            books.Add(new BookInformation("Clean Code", "Robert C. Martin","2008"));
            books.Add(new BookInformation("The Last Lecture", "Randy Pausch","2008"));
            
            throw new System.Exception("Error here!");
            
            return books.ToArray();
        }

    }
}